src/lib/ngsi/EntityIdVector.cpp

```diff
@@ -193,13 +193,18 @@ EntityId* EntityIdVector::get(int ix)
   return vec[ix];
 }
 
+const EntityId* EntityIdVector::get(int ix) const
+{
```

Comment:
Doesn't the operator[] exist for EntityIdVector?
operator[]  should be used instead of get() methods